src/recipes/red-velvet-hot-chocolate.json

```diff
@@ -0,0 +1,63 @@
+{
+    "name": "Red Velvet Hot Chocolate",
+    "description": "Red Velvet Cake in hot chocolate form",
+    "github": "savioxavier",
+    "ingredients": [
+        {
+            "quantity": "2",
+            "measure": "oz",
+            "ingredient": "cream cheese"
+        },
+        {
+            "quantity": "2",
+            "measure": "cups",
+            "ingredient": "milk"
+        },
+        {
+            "quantity": "1",
+            "measure": "cup",
+            "ingredient": "cream"
+        },
+        {
+            "quantity": "2",
+            "measure": "tablespoon",
+            "ingredient": "brown sugar"
+        },
+        {
+            "quantity": "8",
+            "measure": "oz",
+            "ingredient": "white chocolate chips"
+        },
+        {
+            "quantity": "2",
+            "measure": "tablespoon",
+            "ingredient": "unsweetened cocoa powder"
+        },
+        {
+            "quantity": "1",
+            "measure": "tablespoon",
+            "ingredient": "pure vanilla extract"
+        },
+        {
+          "quantity": 1,
```

Comment:
Should be a string.

Reply:
Should be fixed in https://github.com/alfg/opendrinks/pull/1309/commits/7a888eb66952dbd654343277742a9b9857335b36